src/api/comments.ts

```diff
@@ -0,0 +1,37 @@
+export const BASE_URL = 'https://mate.academy/students-api/';
+
+export async function getComments(postId: number) {
+  const response = await fetch(`${BASE_URL}/comments?postId=${postId}`);
+
+  if (!response.ok) {
+    throw new Error(`${response.status} - ${response.statusText}`);
+  }
```

Comment:
Here better to use try catch [construction](https://developer.mozilla.org/ru/docs/Web/JavaScript/Reference/Statements/try...catch). The same is for other async requests.